x-pack/plugins/security_solution/server/plugin.ts

```diff
@@ -161,7 +166,7 @@ export class Plugin implements ISecuritySolutionPlugin {
     initUsageCollectors({
       core,
       kibanaIndex: core.savedObjects.getKibanaIndex(),
-      signalsIndex: config.signalsIndex,
+      signalsIndex: DEFAULT_ALERTS_INDEX,
```

Comment:
NOTE: This will turn into `.alerts-security.alerts*` downstream which I don't think will be a problem and it searches across multiple spaces. This is what the previous behavior was.

Reply:
@marshallmain any user configurations around the naming of the signals index using this `const` might miss? 

I'd remembered a lot of open questions around the index naming and advanced settings back a few months ago so just wanted to double check.

Reply:
This should work, only the `namespace` part of `.alerts-security.alerts-<namespace>` is configurable